### ISSUE TITLE
Add "force" option to baseline

### ISSUE THREAD

**Issue by JacksonBailey** [Cobliteam/cassandra-migrate#31]
It would be nice to be able to use a `--force` flag (or similar) when using the baseline functionality to drop the existing migrations table.

In our environment the migrations table seems to get messed up (or dropped or something) periodically. Why this is happening isn't the concern of this post (and I don't think it's the fault of this tool). To fix it, we can use `baseline` *but I had to first drop the existing migrations table* (`migrate` had been run and it failed on the first script (obviously) because the tables already existed).

**Comment by RyanSattler** [comment]
Yeah sometimes it would be nice to have an option to say "The DB is in the right state, just mark all migrations as correct" (in my case there was a problem when removing a comment from an already-ran migration file).
